plugins/woocommerce/src/Enums/ProductTaxStatus.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Enums;
+
+/**
+ * Enum class for all the product tax statuses.
+ */
+class ProductTaxStatus {
+	/**
+	 * Tax status for products that are taxable.
+	 *
+	 * @var string
+	 */
+	const TAXABLE = 'taxable';
+
+	/**
+	 * Tax status for products that are shipping only.
```

Comment:
This reads a little like it is only used with physical products (which I guess is partly true, but still feels a little misleading to me) ... might this be clearer?

```suggestion
	 * Indicates that only the shipping cost should be taxed, not the product itself.
```